<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>59</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="container_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>68</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Groups :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="group_list">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>28</height>
        </size>
       </property>
       <property name="currentText">
        <string>Select a Group</string>
       </property>
       <item>
        <property name="text">
         <string>Select a Group</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
